src/styles/index.scss

```diff
@@ -1,3 +1,47 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/header';
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+a,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  display: inline-block;
+}
+
+.main {
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
 }
```

Comment:
It's great that you're using media queries to make your layout responsive. However, consider moving them to a separate file for better organization of your styles.